### PR TITLE
[4.1] jooa11y plugin options

### PR DESCRIPTION
**Pull request by brianteeman**
In the initial pull request the removal of some options went too far and stopped the remaining options from working. This resolves that.

This pr also changes the description of the containers to include examples of using classes, divs and roles

Before this PR none of the options were used. Now they are.

### CODE REVIEW

File: plugins/system/jooa11y/jooa11y.php [hunk 1]
```diff
@@ -123,6 +123,17 @@ public function initJooa11y()
 		// Detect the current active language
 		$lang = Factory::getLanguage()->getTag();
 
+
```
Comment (richard67) [inline plugins/system/jooa11y/jooa11y.php]:
```suggestion
```
Remove one of the 2 empty lines to fix PHP CS error in drone.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on d151e21f9905e85f509dfb5fd60a8f5da4366392<br /><br />Without the fix, parameters are not used. With the fix they are.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37196">issues.joomla.org/tracker/joomla-cms/37196</a>.</sub>

File: administrator/language/en-GB/plg_system_jooa11y.ini [hunk 2]
```diff
@@ -5,11 +5,11 @@
 
 PLG_SYSTEM_JOOA11Y="System - Joomla Accessibility Checker"
 PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT="Content Container"
-PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT_DESC="Aria Landmark on the page that will be checked for accessibility. The default setting is <strong>main</strong>."
+PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT_DESC="Landmark on the page that will be checked for accessibility. The default setting is the landmark <strong>main</strong>. Alternatives to landmarks are classes, elements or Aria roles (eg #example, .example, [role=example]."
```
Comment (Quy) [inline administrator/language/en-GB/plg_system_jooa11y.ini]:
```suggestion
PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT_DESC="Landmark on the page that will be checked for accessibility. The default setting is the landmark <strong>main</strong>. Alternatives to landmarks are classes, elements or ARIA roles (eg #example, .example, [role=example])."
```

File: administrator/language/en-GB/plg_system_jooa11y.ini [hunk 3]
```diff
@@ -5,11 +5,11 @@
...
 PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE="Ignore Regions"
 PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE_DESC="Ignore specific regions within the Content Container. Use commas to separate classes or elements (eg #ignore, .ignore)"
 PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT="Readability Container"
-PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT_DESC="Aria Landmark on the page that will be checked for readability. The default setting is <strong>main</strong>."
+PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT_DESC="Landmark on the page that will be checked for readability. The default setting is the landmark <strong>main</strong>. Alternatives to landmarks are classes, elements or Aria roles (eg #example, .example, [role=example]."
```
Comment (Quy) [inline administrator/language/en-GB/plg_system_jooa11y.ini]:
```suggestion
PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT_DESC="Landmark on the page that will be checked for readability. The default setting is the landmark <strong>main</strong>. Alternatives to landmarks are classes, elements or ARIA roles (eg #example, .example, [role=example])."
```

File: administrator/language/en-GB/plg_system_jooa11y.ini [hunk 4]
```diff
@@ -5,11 +5,11 @@
 
 PLG_SYSTEM_JOOA11Y="System - Joomla Accessibility Checker"
 PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT="Content Container"
-PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT_DESC="Aria Landmark on the page that will be checked for accessibility. The default setting is <strong>main</strong>."
+PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT_DESC="Landmark on the page that will be checked for accessibility. The default setting is the landmark <strong>main</strong>. Alternatives to landmarks are classes, elements or Aria roles (eg #example, .example, [role=example]."
 PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE="Ignore Regions"
 PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE_DESC="Ignore specific regions within the Content Container. Use commas to separate classes or elements (eg #ignore, .ignore)"
```
Comment (Quy) [inline administrator/language/en-GB/plg_system_jooa11y.ini]:
```suggestion
PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE_DESC="Ignore specific regions within the Content Container. Use commas to separate classes or elements (eg #ignore, .ignore)."
```

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on a8836e35ffb8ad6f3843e7ea17bb23ab37166e9a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37196">issues.joomla.org/tracker/joomla-cms/37196</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on a8836e35ffb8ad6f3843e7ea17bb23ab37166e9a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37196">issues.joomla.org/tracker/joomla-cms/37196</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37196">issues.joomla.org/tracker/joomla-cms/37196</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
thx

**Pull request merged**